pom.xml

```diff
@@ -143,6 +143,8 @@
 
         <rocksdb-dependency-scope>compile</rocksdb-dependency-scope>
         <lucene-dependency-scope>compile</lucene-dependency-scope>
+
+        <paimon-dependency-scope>compile</paimon-dependency-scope>
```

Comment:
Would it be more appropriate to change the default dependency scope to runtime like before?

Reply:
Change to compile because the scope for parent pom is compile, will change to runtime if this is valid.